web/src/test/java/org/springframework/security/web/DefaultRedirectStrategyTests.java

```diff
@@ -64,4 +71,19 @@ public void contextRelativeShouldThrowExceptionIfURLDoesNotContainContextPath()
 				.isThrownBy(() -> rds.sendRedirect(request, response, "https://redirectme.somewhere.else"));
 	}
 
+	@ParameterizedTest
+	@EnumSource(value = HttpStatus.class, names = { "FOUND", "SEE_OTHER", "TEMPORARY_REDIRECT" })
```

Comment:
> It would also be valuable to assert that the response is not committed, so as to clearly indicate the difference between this approach and calling `sendRedirect`.

Referring to https://github.com/spring-projects/spring-security/pull/12817#discussion_r1307930631, it seems like you ultimately decided you'd prefer the response to be committed in any case. Is that right?